lib/aws/awsconfig.js

```diff
@@ -20,6 +20,73 @@ function AWSConfig() {
     return configService;
   }
 
+  me.enableRule = function(input) {
+    var self = arguments.callee;
+    var configService = me.findService(input);
+    if(input.owner == "CUSTOM_LAMBDA"){
+    var params = {
```

Comment:
This is a common code and isnt only meant when input.owner is CUSTOM_LAMBDA. The input has the relevant values.
The comment "**//Owner: "CUSTOM_LAMBDA"**," is not an inline comment and can be removed.